node/config/config_test.go

```diff
@@ -57,6 +59,7 @@ config:
         max_connections: 10
     grpc:
         address: http://localhost:9090
+    ignore_connect_vote_extension_tx: false
```

Comment:
Can we set this to `true` and ensure that it's able to parse this? Cause the zero value of `bool` is `false`, so the original test will still pass when it can't parse the config correctly.